Project.toml

```diff
@@ -4,15 +4,17 @@ authors = ["Matthew Fishman <mfishman@flatironinstitute.org> and contributors"]
 version = "0.0.1"
 
 [deps]
-ITensorTDVP = "25707e16-a4db-4a07-99d9-4d67b7af0342"
+ITensorMPS = "0d1a4710-d33b-49a5-8f18-73bdf49b47e2"
 ITensors = "9136182c-28ba-11e9-034c-db9fb085ebd5"
+ITensorTDVP = "25707e16-a4db-4a07-99d9-4d67b7af0342"
```

Comment:
Can we remove the `ITensorTDVP` dependency since it is superceded by `ITensorMPS`?